### PR TITLE
update favicon

### PR DESCRIPTION
**Pull request by ania047**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/genezio.svg" />
+    <link rel="icon" type="image/x-icon" href="/apvfavicon.ico" />
```
Comment (cristim67) [inline client/index.html:5]: rename to apv_favicon.

File: client/src/components/Header.component.tsx [hunk 2]
```diff
@@ -120,6 +120,7 @@ export const Header = () => {
             </button>
             <button className="bg-[#1b756f] hover:bg-[#3f9892] translate-x-10 h-[40%] rounded-md text-white text-base text-bold w-[20%]">
               ÎNSCRIERE
+
```
Comment (cristim67) [inline client/src/components/Header.component.tsx:123]: remote this endline.

File: client/src/components/PageWrapper.component.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from "react";
+import { Helmet } from "react-helmet";
+
+interface PageWrapperProps {
+  title: string;
+}
+
+const PageWrapper: React.FC<PageWrapperProps> = ({ title, children }) => {
```
Comment (cristim67) [inline client/src/components/PageWrapper.component.tsx:8]: where you include this?

File: client/index.html (hunk 1 above)
Comment (ania047) [reply client/index.html]: done

File: client/src/components/Header.component.tsx (hunk 2 above)
Comment (ania047) [reply client/src/components/Header.component.tsx]: done

**Pull request merged**